lib/core/src/lib/form/components/widgets/date-time/date-time.widget.ts

```diff
@@ -47,13 +52,13 @@ import { ErrorMessageModel } from '../core/error-message.model';
     imports: [NgIf, TranslateModule, MatFormFieldModule, MatInputModule, MatDatetimepickerModule, ReactiveFormsModule, ErrorWidgetComponent],
     encapsulation: ViewEncapsulation.None
 })
-export class DateTimeWidgetComponent extends WidgetComponent implements OnInit, OnDestroy {
+export class DateTimeWidgetComponent extends WidgetComponent implements OnInit {
     minDate: Date;
     maxDate: Date;
     datetimeInputControl: FormControl<Date> = new FormControl<Date>(null);
 
-    private onDestroy$ = new Subject<void>();
 
+    private destroyRef = inject(DestroyRef);
```

Comment:
when you change it to private readonly as mentioned in one of previous comments then please move it below together with rest private readonly fields